src/otx/algorithms/visual_prompting/tasks/openvino.py

```diff
@@ -335,4 +408,80 @@ def optimize(
         optimization_parameters: Optional[OptimizationParameters] = None,
     ):
         """Optimize function of OpenVINOVisualPromptingTask."""
-        raise NotImplementedError
+        logger.info("Start PTQ optimization")
+        if self.model is None:
+            raise RuntimeError("PTQ optimize failed, model is None")
+
+        if optimization_type is not OptimizationType.POT:
+            raise ValueError("PTQ is the only supported optimization type for OpenVino models")
+
+        dataset = dataset.get_subset(Subset.TRAINING)
+
+        for i, (name, is_encoder) in enumerate(zip(["image_encoder", "decoder"], [True, False]), 1):
+            if name == "decoder":
+                # TODO (sungchul): quantize decoder, too
```

Comment:
Why can't we quantize decoder?

Reply:
When compiling compressed decoder, I have encountered `vector::_M_fill_insert` error.
I think somewhere is needed to fill a value but finding the location seems to take time.
And as you may know, image encoder is very huge than decoder, so inference bottleneck occurred in image encoder. 
(1.84 fps for image encoder and 107.90 fps for decoder on benchmark_app)
So, we decided that it is okay to only compress image encoder.